lms/envs/common.py

```diff
@@ -114,6 +114,15 @@
 
     # for consistency in user-experience, keep the value of the following 3 settings
     # in sync with the corresponding ones in cms/envs/common.py
```

Comment:
1. Would you mind adding a message in the warning about keeping LMS and CMS settings in sync? You could also add a comment in CMS.  Ultimately, we should be able to remove this comment. See https://discuss.openedx.org/t/doc-a-thon-documentation-of-lms-and-cms-common-settings/4149/5?u=robrap.
2. If you use git blame, can you see what the original "following 3 settings" refers to?  Can we update the others as well with the warning and comment?

Reply:
This was marked resolved, but I didn't see most of what was requested.  I'm going to be more explicit.

1. Please remove the following lines (115-116):
```
    # for consistency in user-experience, keep the value of the following 3 settings
    # in sync with the corresponding ones in cms/envs/common.py
```
Note: This comment [originally referred to the settings](https://github.com/edx/edx-platform/blob/1bd6792a080b9cdf63c980ab309db5bbf8743699/cms/envs/common.py#L46-L50) `ENABLE_DISCUSSION_SERVICE`, `ENABLE_TEXTBOOK`, and `ENABLE_STUDENT_NOTES`, the last of which no longer even exists.

2. Please add the following `toggle_warnings` text to `ENABLE_DISCUSSION_SERVICE` and `ENABLE_TEXTBOOK`:
```
For consistency in user-experience, keep the value in sync with the setting of the same name in the CMS.
```